bin/makePsfLibrary.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2015 AURA/LSST
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import pyfits
+import argparse
+import sys
+import os.path
+import numpy
+import lsst.daf.persistence as dafPersist
+import lsst.afw.cameraGeom as cameraGeom
+import lsst.afw.coord as afwCoord
+import lsst.afw.detection as afwDetection
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.afw.table as afwTable
+import lsst.meas.base as measBase
+import lsst.meas.algorithms as measAlg
+
+psfSize = 33
```

Comment:
What is ``psfSize``? We can reasonably conclude it's the size of a PSF, but a comment providing units, justification for choosing this number, etc would help.

As a module-level constant, this should probably be called ``PSF_SIZE``.